DOCUMENTATION.md

```diff
@@ -1,810 +1,780 @@
 
+|Method|Information|
+|:---|:---|
+|`response.links`|Returns a hash linking to the requested notifications (limited to 250)|
+|`response.collection`|Returns an array of the required notifications|
 
-## Get a template by ID
+If you call the `collection` method on this object to return an array, you will then have to call different methods on this array to return the requested information.
 
-#### Method
+If the notification specified in the `older_than` argument is older than 7 days, the client will return an empty `collection` response.
 
-This will return the latest version of the template. Use [getTemplateVersion](#get-a-template-by-id-and-version) to retrieve a specific template version.
+### Error codes
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+If the request is not successful, the client will return a `Notifications::Client::RequestError` containing the relevant error code:
 
-```ruby
-template = client.get_template_by_id(id)
-```
+|error.code|error.message|How to fix|
+|:---|:---|:---|
+|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "bad status is not one of [created, sending, sent, delivered, pending, failed, technical-failure, temporary-failure, permanent-failure, accepted, received]"`<br>`}]`|Contact the Notify team|
+|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "Template type is not one of [sms, email, letter]"`<br>`}]`|Contact the Notify team|
+|`403`|`[{`<br>`"error": "AuthError",`<br>`"message": "Error: Your system clock must be accurate to within 30 seconds"`<br>`}]`|Check your system clock|
+|`403`|`[{`<br>`"error": "AuthError",`<br>`"message": "Invalid token: signature, api token not found"`<br>`}]`|Use the correct API key. Refer to [API keys](/#api-keys) for more information|
 
-</details>
+# Get a template
 
+## Get a template by ID
 
-#### Response
+### Method
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+This will return the latest version of the template.
 
-```Ruby
-template.id         # => uuid for the template
-template.type       # => type of template one of email|sms|letter
-template.created_at # => date and time the template was created
-template.updated_at # => date and time the template was last updated, may be null if version 1
-template.created_by # => email address of the person that created the template
-template.version    # => version of the template
-template.body       # => content of the template
-template.subject    # => subject for email templates, will be empty for other template types
+```ruby
+response = client.get_template_by_id(id)
 ```
 
-Otherwise the client will raise a `Notifications::Client::RequestError`.
+### Arguments
 
-</details>
+#### id (required)
 
+The ID of the template. You can find this by signing into GOV.UK Notify and going to the __Templates__ page.
 
-#### Arguments
+```
+id:'f33517ff-2a88-4f6e-b855-c550268ce08a'
```

Comment:
I think it's clearer if this line just reads `f33517ff-2a88-4f6e-b855-c550268ce08a` (so we don't have the `id:` bit at the beginning). That's because in most cases you use the methods like this `my_method(id: 'my_id')`, but for this particular method you would just use it like this - `my_method('my_id')`. Including `id: ` would give an error.